apps/block_scout_web/assets/js/lib/analytics.js

```diff
@@ -130,6 +130,8 @@ function getEntityId () {
 function getNetwork () {
   const host = window.location.host
   switch (true) {
```

Comment:
interesting i've never seen this done with switch statement setting the switch to true. 

not a value judgement just a oh wow that's possible moment.

Reply:
I generally wouldn't do it but there would be SO MANY if/else statements in this file if I didn't